tests/PHPStan/Analyser/data/bug-7483.php

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types=1);
+
+namespace Bug7483;
```

Comment:
I tried supporting things like

```
function bar() {
	$absolute = '\A';
	$relative = 'A';

	assertType('class-string', ltrim($absolute, '\\'));
	assertType('class-string', ltrim($relative, '\\'));
}
```

but from the extension point of view these strings seem not to be class-strings...?

Reply:
ConstantStringType::isClassString() is just a hint whether the string was created from ::class, so that we don't trigger the autoloading for irrelevant strings, which upsets some custom autoloaders.

Reply:
So you don't need to do anything about it.